### PR TITLE
Create class for error messages

### PR DESCRIPTION
**Pull request by ystoikov**
dev
## ZenHub

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009/issues/ita-social-projects/home/388)

## Summary of issue

Needed to create a class for error messages

## Summary of change

- Add a new ExceptionMessages class that aggregates all error messages from the project
- Replace  all messages in exception parameters
- Add static import to classes with replaced messages

## Testing approach

Not required

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: home-oauth-server/src/main/java/com/softserveinc/ita/homeproject/homeoauthserver/service/OauthServiceImpl.java [hunk 1]
```diff
@@ -16,6 +19,7 @@
 import org.springframework.stereotype.Service;
 
 
+
```
Comment (vladyslavyarets) [inline home-oauth-server/src/main/java/com/softserveinc/ita/homeproject/homeoauthserver/service/OauthServiceImpl.java]: Please, remove this one unnecessary empty line

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/exception/ExceptionMessages.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+    public static final String NOT_FOUND_APARTMENT_ID_MESSAGE = "Apartment with 'id: %d' is not found";
+
+    public static final String NOT_FOUND_HOUSE_WITH_ID_IN_COOPERATION_MESSAGE = "House with id: %d not found "
+        + "in this cooperation";
+
+    public static final String NOT_FOUND_COOPERATION_WITH_ID_MESSAGE = "Can't find cooperation with given ID: %d";
+
+    public static final String NOT_FOUND_HOUSE_WITH_ID_MESSAGE = "House with 'id: %d' is not found";
+
```
Comment (vladyslavyarets) [inline home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/exception/ExceptionMessages.java]: Please, remove this one unnecessary empty line. You don't need it before closing bracket

**Review by vladyslavyarets — CHANGES_REQUESTED** [review]

**Review by vladyslavyarets — APPROVED** [review]

**Comment by vladyslavyarets** [comment]
@MrScors @DzigMS we need your help to handle Sonar Cloud Quality Gate issues

**Review by gendalf-fx — APPROVED** [review]
Nice

**Pull request merged**